dask/dataframe/io/sql.py

```diff
@@ -97,6 +97,9 @@ def read_sql_table(table, uri, index_col, divisions=None, npartitions=None,
     q = sql.select(columns).limit(5).select_from(table)
     head = pd.read_sql(q, engine, **kwargs)
 
+    if head.empty:
+        return from_pandas(head, npartitions=1)
```

Comment:
What would happen before this change?

Reply:
It would raise a value error. It is described in issue #2935. `np.linspace` would fail when determining the divisions.